core/ui/src/main/res/layout/bottomsheet.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:background="@drawable/shape_fill_top_16_rect"
+    android:backgroundTint="@color/white"
+    android:paddingTop="14dp"
+    android:paddingBottom="39dp">
+
+    <ImageView
+        android:id="@+id/bar_bottomsheet"
```

Comment:
P3 :
```suggestion
        android:id="@+id/iv_bottomsheet_bar"
```

Reply:
thank you

Reply:
[038140c](https://github.com/Team-Wable/WABLE-ANDROID/pull/31/commits/038140c2d3f41030a7438ff2d1d48909b49d4de1)